Sunrise/Models/Money.swift

```diff
@@ -9,7 +9,7 @@ struct Money: Mappable {
     // MARK: - Properties
 
     var currencyCode: String?
-    var centAmount: Int?
+    var centAmount: Double?
```

Comment:
I would prefer to keep the `Int`, as this is what the platform will always return.

(On a sidenote, `/ 100` is correct for most western currencies, but e.g. fails for the Japanese Yen (which doesn't have cents) and others. But it's out of scope for now - couldn't find an iOS lib that deals with currencies that far with 5 min of googling).